test/integration/steps/utilities/change_event_builder.py

```diff
@@ -283,4 +275,16 @@ def set_opening_times_change_event(service_type: str) -> ChangeEvent:
             "IsOpen": False,
         }
     )
+    change_event.specified_opening_times.append(
```

Comment:
This would break other tests as the opening times should be marked as illogical if this is received in a change event

Reply:
Change now moved to step def file.